components/cluster-coordinator/org.wso2.carbon.cluster.coordinator/org.wso2.carbon.cluster.coordinator.commons/src/main/java/org/wso2/carbon/cluster/coordinator/commons/configs/CoordinationStrategyConfigUtil.java

```diff
@@ -22,10 +22,42 @@
  */
 public class CoordinationStrategyConfigUtil {
 
+    /**
+     * Variable to hold DataSource configurations.
+     */
+    private Map<String, String> dataSourceConfig;
+
+    /**
+     * Variable to hold RDBMS based coordination configurations.
+     */
     private Map<String, Integer> rdbmsConfig;
+
+    /**
+     * Variable to hold Zookeeper based coordination configurations.
+     */
     private Map<String, String> zookeeperConfig;
+
+    /**
+     * Variable to hold the coordination strategy.
+     */
     private String strategy;
 
+    /**
+     * Method to retrieve DataSource configurations specified in coordinatorConfig.yaml file.
+     * @return  map of DataSource configurations
+     */
+    public Map<String, String> getDataSourceConfig() {
+        return this.dataSourceConfig;
+    }
+
+    /**
+     * Method to set the DataSource configurations.
```

Comment:
Need newline after method description.

Reply:
This code segment is obsolete with later changes.